### ISSUE TITLE
Not able use multiple styles

### ISSUE THREAD

**Issue by twaghmiles** [selemxmn/ngx-print#10]
**Describe the bug**
If we give multiple styles to an element, no style is applied to the element.

**To Reproduce**
Add one more style to `h1` tag in `[printStyle]`. e.g. `[printStyle]="{h1 : {'color': 'red', 'border': '1px solid'}, h2 : {'border': 'solid 1px'}}"`

**Expected behavior**
`[printStyle]` should accept multiple styles for an element.

**Screenshots**
[stackblitz](https://stackblitz.com/edit/demo-ngx-print-dndfzz)

**Desktop (please complete the following information):**
 - Windows 10
 - chrome V71, Firefox V63

**Comment by twaghmiles** [comment]
@selemxmn After updating to `1.1.1`, It says `Cannot find module 'ngx-print'.` Downgrading to `1.1.0` is also giving same error.

**Comment by selemxmn** [comment]
@twaghmiles , That is so weird, it is most likely an npm-registry problem, I found similar problems for other npm packages (i.e: [this one](https://github.com/npm/npm/issues/5082)), I added simple function (not more than 20 caracters ) in my code, that cannot lead to such problem for sure. I'll republish today before 00:00 UTC. Thanks indeed for the alert, I only tested the fix  locally, thus I did not pay attention after releasing and retesting the deployed patch, thinking there'll be no difference. I must apologize, stay tuned tonight.
